docs/manage/CLI.md

```diff
@@ -190,6 +193,107 @@ The `logout` command ends an authenticated CLI session
 viam logout
 ```
 
+### module
+
+The `module` command allows to you to manage custom modules.
+This includes:
+
+* Creating a new custom modular resource on your local filesystem
+* Updating an existing module with new changes
+* Uploading a new module to the Viam Registry
+* Updating an existing module in the Viam Registry
+
+```sh {class="command-line" data-prompt="$"}
+viam module create --organization=<org name> --name <module-id>
+viam module update --organization=<org name> --name <module-id>
```

Comment:
note: for a public module, setting public_namespace is required

Reply:
Should include instructions for how to fill out the meta.json: 

A meta.json file was generated in your current working directory. Open that file and edit it.
A description field is required
By default the module is created with private visibility. Set the visibility to public if you are planning on making the module public during this update. The update command can be run again later to make the module public if that is desired.
A url is not required but recommended. Set this to the url of the github repository containing the source code of the module.
A list of models is required to contain at least one model. The model and namespace fields are required to be in the triple format. For example:

"models": [
    {
      "api": "rdk:component:generic",
      "model": "michaellee1019:audio_output:play_file"
    },
]

Reply:
Started leaving notes -- but overall, these instructions have the latest up to date info: https://docs.google.com/document/d/1HSOV6ncMv5QkpdWOqp-f_RnUxHz4QnrnchjLLjN69MQ/edit

can you take a stab at updating the CLI documentation with this info?